viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -47,24 +47,34 @@ namespace viam {
 
 static const int checkForShutdownIntervalMicroseconds = 1e5;
 
-// This RGB value represents "no input" from a Cario pained map
-static const unsigned char defaultCairosEmptyPaintedSlice = 102;
 static const int jpegQuality = 50;
 // Byte limit on GRPC, used to help determine sampling skip_count
 static const int maximumGRPCByteLimit = 32 * 1024 * 1024;
 // Byte limit for chunks on GRPC, used for streaming apis
-static const int maximumGRPCByteChunkSize = 64 * 1024;
-// Coeffient to adjust the skip count for the PCD to ensure the file is within
-// grpc limitations. Increase the value if you expect dense feature-rich maps
-static const int samplingFactor = 1;
-// Conversion to number of bytes used in colored PCD encoding
-static const int pixelBytetoPCDByte = 16 / 4;
+static const int maximumGRPCByteChunkSize = 1 * 1024 * 1024;
+
 // Quaternion to rotate axes to the XZ plane
 static const Eigen::Quaterniond pcdRotation(0.7071068, -0.7071068, 0, 0);
 // Static offset quaternion, so orientation matches physical intuition.
 // This will result in rotations occurring within the y axis to match 2D mapping
 // in the XZ plane
 static const Eigen::Quaterniond pcdOffsetRotation(0.7071068, 0.7071068, 0, 0);
+
+// resolution defines the area in meters that each pixel represent. This
+// is used to draw the jpeg map and in so doing defines the resolution of
+// the outputted PCD
+const double resolution = 0.05;
+// Number of bytes in a pixel
+const int bytesPerPixel = 4;
+// Color channel (RGBA) used to determine probability of point
+// - 0 is the R channel
+// - 1 is the B channel
+// - 2 is the G channel
+// - 3 is the A channel
+const int probabilityColorChannel = 2;
+// This RGB value represents "no input" from a Cario pained map
```

Comment:
[q] What does "no input" mean? What does the value of `102` represent? One of the RGB values?

Reply:
no input refers to pixels that the painter has not adjusted based on the submaps (i.e. the default background).

102 is the default RGB value each pixel will have. rather than starting at 0. This 102 is also used in the probability calculation. A pixel that has all 3 channels as the default value([102,102,102]) will result in the grey pixels outside of the map.

Let me know if the adjustment to the comment makes that clear or if you'd like something more